app/process/handler/crypto-transfer/crypto-transfer.go

```diff
@@ -146,6 +149,7 @@ func (cth *CryptoTransferHandler) Handle(payload []byte) {
 	encodedData, err := ethhelper.EncodeData(&ctm)
 	if err != nil {
 		cth.logger.Errorf("Failed to encode data for TransactionID [%s]. Error [%s].", ctm.TransactionId, err)
+		return
```

Comment:
With the addition of this line, the status of processed tx above will never be updated to f.e. `StatusInsufficientFee`